docs/src/pages/components/data-grid/rows/rows.md

```diff
@@ -36,6 +36,15 @@ In addition, the area in which the callback provided to the `onRowsScrollEnd` is
 
 {{"demo": "pages/components/data-grid/rows/InfiniteLoadingGrid.js", "bg": "inline", "disableAd": true}}
 
+### Server-side infinite loading [<span class="pro"></span>](https://material-ui.com/store/items/material-ui-pro/)
+
+By default, infinite loading works on the client-side.
+To switch it to server-side, the `rowCount` needs to be set and the number of initially loaded rows needs to be less than the `rowCount` value. In addition, you need to handle the `onFetchRows` callback to fetch the rows for the corresponding index. Finally, you need to use the `apiRef.current.insertRows()` to tell the DataGrid where to insert the newly fetched rows.
+
+Lastly, in order for the filtering and sorting to work you need to set their modes to `server`. You can find out more information about how to do that [here](https://material-ui.com/components/data-grid/filtering/#server-side-filter) and [here](https://material-ui.com/components/data-grid/sorting/#server-side-sorting).
```

Comment:
Relative URLs so it still works once we version it.

```suggestion
Lastly, in order for the filtering and sorting to work you need to set their modes to `server`.
You can find out more information about how to do that [here](/components/data-grid/filtering/#server-side-filter) and [here](/components/data-grid/sorting/#server-side-sorting).
```

Reply:
We **can't** keep "here" as a label for a link. It's the most common pitfall with a11y for links, for instance, the first link/header doing a quick search: https://www.a11yproject.com/posts/2019-02-15-creating-valid-and-accessible-links/

Reply:
How about like this:

```
You can find out more information about how to do that on the [server side filter page](/components/data-grid/filtering/#server-side-filter) and on the [server side sorting page](/components/data-grid/sorting/#server-side-sorting).

Reply:
With `server-side` 👍  (a writing style)